rest/models/markets.go

```diff
@@ -11,6 +12,13 @@ type GetMarketHolidaysParams struct{}
 // GetMarketHolidaysResponse is the response returned by the GetMarketHolidays method.
 type GetMarketHolidaysResponse []MarketHoliday
 
+// todo: this endpoint is unlikely to ever have params so should we delete this type?
+// GetMarketStatusParams is the set of parameters for the GetMarketStatus method.
+type GetMarketStatusParams struct{}
```

Comment:
i'm thinking we should delete these after a little more thought. wanna remove this and the market holiday params?

Reply:
Sure I'll remove them